sherlock/resources/data.json

```diff
@@ -1378,6 +1386,14 @@
     "urlMain": "https://www.needrom.com/",
     "username_claimed": "needrom"
   },
+  "Neocities": {
```

Comment:
Neocities has info on valid usernames. With this, we can craft a regex:

```
^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$
```

Could you include this as well?